### DISCUSSION TITLE
Herd for Linux?

### DISCUSSION THREAD

**Discussion by spam-n-eggs** [beyondcode/herd-community#646] [Ideas]
Is there a possibility that there will ever be a Herd for Linux?  
@cpriego released a linux port of valet here: https://github.com/cpriego/valet-linux, so it gives me hope that it could be possible with Herd, too?

**Comment by amjeed-ay** [comment]
Linux version needed

**Comment by vdvcoder** [comment]
Hope there comes a Linux version soon. It's one of the only things I want since i moved from macOs to Linux. I miss it here.

**Comment by geodro** [comment]
Check this out, it is Herd like and works on Linux 

https://github.com/lerd-env/lerd

**Comment by ahosker** [comment]
There is a version of Laravel Herd for Linux https://php.new/ there is no documentation (yet?) on the main site tho.

**Reply by kasperhartwich** [reply]
php.new does not install Herd, but php, composer and so on.
You can see the install script here: https://php.new/install/linux

**Reply by ahosker** [reply]
Sure, not as we know it. However, it's the same team, and the script's internal name and install directory is `herd-lite`.

**Reply by kasperhartwich** [reply]
It's just the compiled php and composer binaries they retrieve. 
It does not make it Herd.
